src/components/Main.jsx

```diff
@@ -0,0 +1,67 @@
+import style from './Main.module.css';
+import Person from '../../images/illustration-working.svg?react';
+import Brand from '../../images/icon-brand-recognition.svg?react';
+import Detailed from '../../images/icon-detailed-records.svg?react';
+import Fully from '../../images/icon-fully-customizable.svg?react';
+import { InfoBlock } from '../InfoBlock';
+
+function Main() {
+  return (
+    <div className={style.main}>
+      <div className={style.more_img}>
+        <section className={style.more}>
+          <h1 className={style.more_then}>More than just shorter links</h1>
+          <p className={style.p}>
+            Build your brand&apos;s and get detailed insights on how your links are performing
+          </p>
+          <button className={style.getStarted}>Get Started</button>
+        </section>
```

Comment:
Still to many markup details for one component.

Let's build next components:
- Button, Input, Title, Text (as UI Kit components)
- Hero (the first section with Title, Text and Button)
- Shortner (with Input and Button inside)
- Feature (with Title and Text) and FeatureList
- InfoBlock (with Title, Text and FeatureList)
- CallToAction (with Title and Button)